docs/measure-kernel-mcdc.md

```diff
@@ -9,8 +9,8 @@
 
 - The following instructions are tested with:
     - Architecture: x86_64
-    - Distro: Ubuntu 22.04
-    - Kernel: 5.15.0-86-generic
+    - Distro: Ubuntu 22.04 and Debian trixie/sid
+    - Kernel: 5.15.0 to 5.15.153
```

Comment:
Here I actually meant kernel version of the *host*, which again should not make much difference and is just informative.
I will revert the text to the setup I have been using (5.15.0-86-generic) and you are welcome to add yours (the Debian one).